packages/replay-internal/src/util/createPerformanceEntries.ts

```diff
@@ -225,13 +229,24 @@ export function getInteractionToNextPaint(metric: Metric): ReplayPerformanceEntr
 export function getWebVital(
   metric: Metric,
   name: string,
-  node: Node | undefined,
+  node: Node | Node[] | undefined,
```

Comment:
Is `getWebVital` a public function? It might be better to change the parameters so that it accepts `Node[] | undefined`.